.eslintignore

```diff
@@ -2,4 +2,4 @@
 /config/
 /dist/
 /*.js
-/__tests__/unit/coverage/
+/node_modules/**
```

Comment:
please keep the coverage line too in case someone ran it locally and the coverage folder was generated